[12-11-2020:14:15:45] p14108 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\websocket.py:85} ERROR - WEBSOCKET | Connection Failure:: Cannot connect to host 127.0.0.1:2333 ssl:default [The remote computer refused the network connection]
[12-11-2020:14:15:45] p14108 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:138} INFO - NODE | MAIN connected:: MAIN | europe | (Shard: None)
[12-11-2020:14:15:45] p14108 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\client.py:430} INFO - CLIENT | New node initiated:: MAIN | europe | (Shard: None) 
[12-11-2020:14:15:53] p14108 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\player.py:268} INFO - PLAYER | Connected to voice channel:: 730260467733233778
[12-11-2020:14:19:08] p14108 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\player.py:279} INFO - PLAYER | Disconnected from voice channel:: 730260467733233778
[12-11-2020:14:19:14] p14108 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\player.py:268} INFO - PLAYER | Connected to voice channel:: 730260467733233778
[12-11-2020:14:31:08] p13408 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\websocket.py:85} ERROR - WEBSOCKET | Connection Failure:: Cannot connect to host 127.0.0.1:2333 ssl:default [The remote computer refused the network connection]
[12-11-2020:14:31:08] p13408 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:138} INFO - NODE | MAIN connected:: MAIN | europe | (Shard: None)
[12-11-2020:14:31:08] p13408 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\client.py:430} INFO - CLIENT | New node initiated:: MAIN | europe | (Shard: None) 
[12-11-2020:14:31:15] p13408 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\player.py:268} INFO - PLAYER | Connected to voice channel:: 730260467733233778
[12-11-2020:14:44:12] p12560 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\websocket.py:85} ERROR - WEBSOCKET | Connection Failure:: Cannot connect to host 127.0.0.1:2333 ssl:default [The remote computer refused the network connection]
[12-11-2020:14:44:12] p12560 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:138} INFO - NODE | MAIN connected:: MAIN | europe | (Shard: None)
[12-11-2020:14:44:12] p12560 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\client.py:430} INFO - CLIENT | New node initiated:: MAIN | europe | (Shard: None) 
[12-11-2020:14:44:28] p12560 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\player.py:268} INFO - PLAYER | Connected to voice channel:: 730260467733233778
[12-11-2020:14:49:24] p12924 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\websocket.py:85} ERROR - WEBSOCKET | Connection Failure:: Cannot connect to host 127.0.0.1:2333 ssl:default [The remote computer refused the network connection]
[12-11-2020:14:49:24] p12924 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:138} INFO - NODE | MAIN connected:: MAIN | europe | (Shard: None)
[12-11-2020:14:49:24] p12924 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\client.py:430} INFO - CLIENT | New node initiated:: MAIN | europe | (Shard: None) 
[12-11-2020:14:51:32] p12924 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\player.py:279} INFO - PLAYER | Disconnected from voice channel:: None
[12-11-2020:14:54:17] p10912 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\websocket.py:85} ERROR - WEBSOCKET | Connection Failure:: Cannot connect to host 127.0.0.1:2333 ssl:default [The remote computer refused the network connection]
[12-11-2020:14:54:17] p10912 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:138} INFO - NODE | MAIN connected:: MAIN | europe | (Shard: None)
[12-11-2020:14:54:17] p10912 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\client.py:430} INFO - CLIENT | New node initiated:: MAIN | europe | (Shard: None) 
[12-11-2020:15:27:36] p14784 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:138} INFO - NODE | MAIN connected:: MAIN | europe | (Shard: None)
[12-11-2020:15:27:36] p14784 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\client.py:430} INFO - CLIENT | New node initiated:: MAIN | europe | (Shard: None) 
[12-11-2020:15:28:24] p14784 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\player.py:268} INFO - PLAYER | Connected to voice channel:: 730260467733233778
[12-11-2020:15:28:41] p14784 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackStartEvent> (MAIN | europe | (Shard: None))
[12-11-2020:15:30:46] p14784 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:183} INFO - REST | No tracks with query <https://open.spotify.com/track/5QTKooEvwEXuXJfc1bPJ61?si=efqL4JVFTgqAMwG1hGKl0A> found.
[12-11-2020:15:31:21] p14784 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackEndEvent> (MAIN | europe | (Shard: None))
[12-11-2020:15:31:32] p14784 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackStartEvent> (MAIN | europe | (Shard: None))
[12-11-2020:15:34:26] p14784 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackEndEvent> (MAIN | europe | (Shard: None))
[12-11-2020:15:34:59] p14784 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackStartEvent> (MAIN | europe | (Shard: None))
[12-11-2020:15:38:30] p14784 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackEndEvent> (MAIN | europe | (Shard: None))
[12-11-2020:15:38:30] p14784 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackStartEvent> (MAIN | europe | (Shard: None))
[12-11-2020:15:41:15] p14784 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackEndEvent> (MAIN | europe | (Shard: None))
[12-11-2020:15:41:15] p14784 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackStartEvent> (MAIN | europe | (Shard: None))
[12-11-2020:15:43:19] p14784 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackEndEvent> (MAIN | europe | (Shard: None))
[12-11-2020:15:45:01] p14784 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\websocket.py:112} WARNING - 
WEBSOCKET | Connection closed:: Retrying connection in <9.559059942634903> seconds

[13-11-2020:07:37:50] p5940 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\requests\packages\urllib3\connectionpool.py:757} INFO - Starting new HTTPS connection (1): www.instagram.com
[13-11-2020:07:37:52] p5940 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\requests\packages\urllib3\connectionpool.py:757} INFO - Starting new HTTPS connection (1): www.instagram.com
[13-11-2020:07:37:53] p5940 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\requests\packages\urllib3\connectionpool.py:757} INFO - Starting new HTTPS connection (1): discord.com
[13-11-2020:07:37:59] p5940 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\websocket.py:85} ERROR - WEBSOCKET | Connection Failure:: Cannot connect to host 127.0.0.1:2333 ssl:default [The remote computer refused the network connection]
[13-11-2020:07:38:00] p5940 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:138} INFO - NODE | MAIN connected:: MAIN | europe | (Shard: None)
[13-11-2020:07:38:00] p5940 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\client.py:430} INFO - CLIENT | New node initiated:: MAIN | europe | (Shard: None) 
[13-11-2020:07:39:44] p4660 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\requests\packages\urllib3\connectionpool.py:757} INFO - Starting new HTTPS connection (1): www.instagram.com
[13-11-2020:07:39:51] p4660 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\websocket.py:85} ERROR - WEBSOCKET | Connection Failure:: Cannot connect to host 127.0.0.1:2333 ssl:default [The remote computer refused the network connection]
[13-11-2020:07:39:51] p4660 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:138} INFO - NODE | MAIN connected:: MAIN | europe | (Shard: None)
[13-11-2020:07:39:51] p4660 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\client.py:430} INFO - CLIENT | New node initiated:: MAIN | europe | (Shard: None) 
[13-11-2020:07:47:34] p5528 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\requests\packages\urllib3\connectionpool.py:757} INFO - Starting new HTTPS connection (1): www.instagram.com
[13-11-2020:07:47:41] p5528 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\websocket.py:85} ERROR - WEBSOCKET | Connection Failure:: Cannot connect to host 127.0.0.1:2333 ssl:default [The remote computer refused the network connection]
[13-11-2020:07:47:41] p5528 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:138} INFO - NODE | MAIN connected:: MAIN | europe | (Shard: None)
[13-11-2020:07:47:41] p5528 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\client.py:430} INFO - CLIENT | New node initiated:: MAIN | europe | (Shard: None) 
[13-11-2020:07:47:58] p11064 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\requests\packages\urllib3\connectionpool.py:757} INFO - Starting new HTTPS connection (1): www.instagram.com
[13-11-2020:07:48:05] p11064 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\websocket.py:85} ERROR - WEBSOCKET | Connection Failure:: Cannot connect to host 127.0.0.1:2333 ssl:default [The remote computer refused the network connection]
[13-11-2020:07:48:05] p11064 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:138} INFO - NODE | MAIN connected:: MAIN | europe | (Shard: None)
[13-11-2020:07:48:05] p11064 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\client.py:430} INFO - CLIENT | New node initiated:: MAIN | europe | (Shard: None) 
[13-11-2020:08:50:56] p4664 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\websocket.py:85} ERROR - WEBSOCKET | Connection Failure:: Cannot connect to host 127.0.0.1:2333 ssl:default [The remote computer refused the network connection]
[13-11-2020:08:50:56] p4664 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:138} INFO - NODE | MAIN connected:: MAIN | europe | (Shard: None)
[13-11-2020:08:50:56] p4664 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\client.py:430} INFO - CLIENT | New node initiated:: MAIN | europe | (Shard: None) 
[13-11-2020:08:52:00] p4664 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\player.py:268} INFO - PLAYER | Connected to voice channel:: 776807594085056563
[13-11-2020:08:52:21] p4664 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\player.py:279} INFO - PLAYER | Disconnected from voice channel:: 776807594085056563
[13-11-2020:08:55:37] p10668 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:138} INFO - NODE | MAIN connected:: MAIN | europe | (Shard: None)
[13-11-2020:08:55:37] p10668 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\client.py:430} INFO - CLIENT | New node initiated:: MAIN | europe | (Shard: None) 
[13-11-2020:08:58:06] p10668 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\player.py:268} INFO - PLAYER | Connected to voice channel:: 776807594085056563
[13-11-2020:09:05:49] p10668 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackStartEvent> (MAIN | europe | (Shard: None))
[13-11-2020:09:14:01] p10288 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:138} INFO - NODE | MAIN connected:: MAIN | europe | (Shard: None)
[13-11-2020:09:14:01] p10288 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\client.py:430} INFO - CLIENT | New node initiated:: MAIN | europe | (Shard: None) 
[13-11-2020:09:14:07] p10288 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\player.py:268} INFO - PLAYER | Connected to voice channel:: 776807594085056563
[13-11-2020:09:14:24] p10288 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackStartEvent> (MAIN | europe | (Shard: None))
[13-11-2020:09:15:20] p14184 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:138} INFO - NODE | MAIN connected:: MAIN | europe | (Shard: None)
[13-11-2020:09:15:20] p14184 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\client.py:430} INFO - CLIENT | New node initiated:: MAIN | europe | (Shard: None) 
[13-11-2020:09:15:23] p14184 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\player.py:268} INFO - PLAYER | Connected to voice channel:: 776807594085056563
[13-11-2020:09:15:40] p14184 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackStartEvent> (MAIN | europe | (Shard: None))
[13-11-2020:09:16:34] p14184 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\player.py:279} INFO - PLAYER | Disconnected from voice channel:: 776807594085056563
[13-11-2020:09:16:45] p6588 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:138} INFO - NODE | MAIN connected:: MAIN | europe | (Shard: None)
[13-11-2020:09:16:45] p6588 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\client.py:430} INFO - CLIENT | New node initiated:: MAIN | europe | (Shard: None) 
[13-11-2020:09:16:49] p6588 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\player.py:268} INFO - PLAYER | Connected to voice channel:: 776807594085056563
[13-11-2020:09:16:59] p6588 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackStartEvent> (MAIN | europe | (Shard: None))
[13-11-2020:09:17:25] p6588 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackEndEvent> (MAIN | europe | (Shard: None))
[13-11-2020:09:17:30] p6588 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\player.py:279} INFO - PLAYER | Disconnected from voice channel:: 776807594085056563
[13-11-2020:09:17:31] p6588 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\player.py:279} INFO - PLAYER | Disconnected from voice channel:: None
[13-11-2020:09:45:54] p6588 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\player.py:268} INFO - PLAYER | Connected to voice channel:: 776807459373056035
[13-11-2020:09:46:14] p6588 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackStartEvent> (MAIN | europe | (Shard: None))
[13-11-2020:09:47:27] p6588 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackEndEvent> (MAIN | europe | (Shard: None))
[13-11-2020:09:47:35] p6588 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\player.py:279} INFO - PLAYER | Disconnected from voice channel:: 776807459373056035
[13-11-2020:09:47:45] p6588 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\player.py:279} INFO - PLAYER | Disconnected from voice channel:: None
[13-11-2020:10:43:55] p6588 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\player.py:268} INFO - PLAYER | Connected to voice channel:: 776807594085056563
[13-11-2020:10:44:07] p6588 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackStartEvent> (MAIN | europe | (Shard: None))
[13-11-2020:10:47:59] p6588 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackEndEvent> (MAIN | europe | (Shard: None))
[13-11-2020:10:47:59] p6588 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackStartEvent> (MAIN | europe | (Shard: None))
[13-11-2020:11:33:13] p6588 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackEndEvent> (MAIN | europe | (Shard: None))
[13-11-2020:11:33:13] p6588 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackStartEvent> (MAIN | europe | (Shard: None))
[13-11-2020:11:38:42] p6588 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackEndEvent> (MAIN | europe | (Shard: None))
[13-11-2020:11:38:42] p6588 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackStartEvent> (MAIN | europe | (Shard: None))
[13-11-2020:11:42:28] p13572 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:138} INFO - NODE | MAIN connected:: MAIN | europe | (Shard: None)
[13-11-2020:11:42:28] p13572 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\client.py:430} INFO - CLIENT | New node initiated:: MAIN | europe | (Shard: None) 
[13-11-2020:11:43:02] p2728 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:138} INFO - NODE | MAIN connected:: MAIN | europe | (Shard: None)
[13-11-2020:11:43:02] p2728 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\client.py:430} INFO - CLIENT | New node initiated:: MAIN | europe | (Shard: None) 
[13-11-2020:11:46:35] p6756 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:138} INFO - NODE | MAIN connected:: MAIN | europe | (Shard: None)
[13-11-2020:11:46:35] p6756 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\client.py:430} INFO - CLIENT | New node initiated:: MAIN | europe | (Shard: None) 
[13-11-2020:11:57:17] p6756 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\player.py:279} INFO - PLAYER | Disconnected from voice channel:: None
[13-11-2020:12:11:31] p6756 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\player.py:268} INFO - PLAYER | Connected to voice channel:: 776807594085056563
[13-11-2020:12:11:44] p6756 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackStartEvent> (MAIN | europe | (Shard: None))
[13-11-2020:12:18:08] p6756 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackEndEvent> (MAIN | europe | (Shard: None))
[13-11-2020:12:18:42] p6756 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackStartEvent> (MAIN | europe | (Shard: None))
[13-11-2020:12:19:01] p6756 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackEndEvent> (MAIN | europe | (Shard: None))
[13-11-2020:12:19:04] p6756 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\websocket.py:112} WARNING - 
WEBSOCKET | Connection closed:: Retrying connection in <6.325963899516139> seconds

[13-11-2020:12:19:11] p6756 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\websocket.py:112} WARNING - 
WEBSOCKET | Connection closed:: Retrying connection in <13.476617370448151> seconds

[13-11-2020:12:19:22] p6612 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:138} INFO - NODE | MAIN connected:: MAIN | europe | (Shard: None)
[13-11-2020:12:19:22] p6612 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\client.py:430} INFO - CLIENT | New node initiated:: MAIN | europe | (Shard: None) 
[13-11-2020:12:19:29] p6612 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\player.py:268} INFO - PLAYER | Connected to voice channel:: 776807594085056563
[13-11-2020:12:19:40] p6612 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackStartEvent> (MAIN | europe | (Shard: None))
[13-11-2020:12:23:28] p6612 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackEndEvent> (MAIN | europe | (Shard: None))
[13-11-2020:12:23:29] p6612 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\player.py:279} INFO - PLAYER | Disconnected from voice channel:: 776807594085056563
[13-11-2020:12:23:32] p6612 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\player.py:279} INFO - PLAYER | Disconnected from voice channel:: None
[17-11-2020:08:35:23] p1680 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\asyncio\base_events.py:1707} ERROR - Exception in callback _ProactorBasePipeTransport._call_connection_lost(None)
handle: <Handle _ProactorBasePipeTransport._call_connection_lost(None)>
Traceback (most recent call last):
  File "C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\asyncio\events.py", line 81, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\asyncio\proactor_events.py", line 162, in _call_connection_lost
    self._sock.shutdown(socket.SHUT_RDWR)
ConnectionResetError: [WinError 10054] An existing connection was forcibly closed by the remote host
[17-11-2020:08:35:23] p1680 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:138} INFO - NODE | MAIN connected:: MAIN | europe | (Shard: None)
[17-11-2020:08:35:23] p1680 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\client.py:430} INFO - CLIENT | New node initiated:: MAIN | europe | (Shard: None) 
[17-11-2020:08:35:40] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:138} INFO - NODE | MAIN connected:: MAIN | europe | (Shard: None)
[17-11-2020:08:35:40] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\client.py:430} INFO - CLIENT | New node initiated:: MAIN | europe | (Shard: None) 
[17-11-2020:08:35:46] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\player.py:268} INFO - PLAYER | Connected to voice channel:: 776807594085056563
[17-11-2020:08:36:00] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackStartEvent> (MAIN | europe | (Shard: None))
[17-11-2020:08:39:00] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackEndEvent> (MAIN | europe | (Shard: None))
[17-11-2020:08:40:00] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackStartEvent> (MAIN | europe | (Shard: None))
[17-11-2020:08:42:55] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackEndEvent> (MAIN | europe | (Shard: None))
[17-11-2020:08:48:06] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackStartEvent> (MAIN | europe | (Shard: None))
[17-11-2020:08:50:54] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackEndEvent> (MAIN | europe | (Shard: None))
[17-11-2020:08:51:34] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackStartEvent> (MAIN | europe | (Shard: None))
[17-11-2020:08:53:47] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackEndEvent> (MAIN | europe | (Shard: None))
[17-11-2020:08:58:08] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackStartEvent> (MAIN | europe | (Shard: None))
[17-11-2020:09:00:46] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackEndEvent> (MAIN | europe | (Shard: None))
[17-11-2020:09:06:28] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackStartEvent> (MAIN | europe | (Shard: None))
[17-11-2020:09:08:40] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\player.py:279} INFO - PLAYER | Disconnected from voice channel:: 776807594085056563
[17-11-2020:15:35:08] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\player.py:268} INFO - PLAYER | Connected to voice channel:: 776807594085056563
[17-11-2020:15:35:11] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackStartEvent> (MAIN | europe | (Shard: None))
[17-11-2020:15:38:12] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackEndEvent> (MAIN | europe | (Shard: None))
[17-11-2020:15:38:25] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackStartEvent> (MAIN | europe | (Shard: None))
[17-11-2020:15:41:03] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackEndEvent> (MAIN | europe | (Shard: None))
[17-11-2020:15:41:03] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackStartEvent> (MAIN | europe | (Shard: None))
[17-11-2020:15:42:15] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:183} INFO - REST | No tracks with query <https://www.youtube.com/watchv=EE2ZhMg_uzc&list=RDEE2ZhMg_uzc&start_radio=1> found.
[17-11-2020:15:54:57] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackEndEvent> (MAIN | europe | (Shard: None))
[17-11-2020:15:54:57] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackStartEvent> (MAIN | europe | (Shard: None))
[17-11-2020:15:57:35] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackEndEvent> (MAIN | europe | (Shard: None))
[17-11-2020:15:57:35] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackStartEvent> (MAIN | europe | (Shard: None))
[17-11-2020:15:58:17] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackEndEvent> (MAIN | europe | (Shard: None))
[17-11-2020:15:58:17] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackStartEvent> (MAIN | europe | (Shard: None))
[17-11-2020:15:58:22] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackEndEvent> (MAIN | europe | (Shard: None))
[17-11-2020:15:58:22] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackStartEvent> (MAIN | europe | (Shard: None))
[17-11-2020:15:58:28] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackEndEvent> (MAIN | europe | (Shard: None))
[17-11-2020:15:58:40] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackStartEvent> (MAIN | europe | (Shard: None))
[17-11-2020:16:02:05] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackEndEvent> (MAIN | europe | (Shard: None))
[17-11-2020:16:02:54] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\player.py:279} INFO - PLAYER | Disconnected from voice channel:: 776807594085056563
[17-11-2020:16:02:56] p10224 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\player.py:279} INFO - PLAYER | Disconnected from voice channel:: None
[19-11-2020:08:22:34] p840 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:138} INFO - NODE | MAIN connected:: MAIN | europe | (Shard: None)
[19-11-2020:08:22:34] p840 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\client.py:430} INFO - CLIENT | New node initiated:: MAIN | europe | (Shard: None) 
[19-11-2020:13:48:45] p15264 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\websocket.py:85} ERROR - WEBSOCKET | Connection Failure:: Cannot connect to host 127.0.0.1:2333 ssl:default [The remote computer refused the network connection]
[19-11-2020:13:48:45] p15264 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:138} INFO - NODE | MAIN connected:: MAIN | europe | (Shard: None)
[19-11-2020:13:48:45] p15264 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\client.py:430} INFO - CLIENT | New node initiated:: MAIN | europe | (Shard: None) 
[24-11-2020:18:54:56] p7624 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:138} INFO - NODE | MAIN connected:: MAIN | europe | (Shard: None)
[24-11-2020:18:54:56] p7624 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\client.py:430} INFO - CLIENT | New node initiated:: MAIN | europe | (Shard: None) 
[24-11-2020:18:55:18] p7624 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\player.py:268} INFO - PLAYER | Connected to voice channel:: 776807459373056035
[24-11-2020:18:55:31] p7624 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackStartEvent> (MAIN | europe | (Shard: None))
[24-11-2020:18:58:57] p7624 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackEndEvent> (MAIN | europe | (Shard: None))
[24-11-2020:18:59:34] p7624 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <WebsocketClosedEvent> (MAIN | europe | (Shard: None))
[24-11-2020:19:09:40] p4840 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:138} INFO - NODE | MAIN connected:: MAIN | europe | (Shard: None)
[24-11-2020:19:09:40] p4840 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\client.py:430} INFO - CLIENT | New node initiated:: MAIN | europe | (Shard: None) 
[24-11-2020:19:09:52] p4840 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\player.py:268} INFO - PLAYER | Connected to voice channel:: 776807459373056035
[24-11-2020:19:10:36] p4840 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackStartEvent> (MAIN | europe | (Shard: None))
[24-11-2020:19:13:41] p4840 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackEndEvent> (MAIN | europe | (Shard: None))
[24-11-2020:19:14:23] p4840 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:247} INFO - NODE | Event dispatched:: <TrackStartEvent> (MAIN | europe | (Shard: None))
[24-11-2020:19:15:46] p4840 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\player.py:279} INFO - PLAYER | Disconnected from voice channel:: 776807459373056035
[24-11-2020:19:16:04] p15868 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:138} INFO - NODE | MAIN connected:: MAIN | europe | (Shard: None)
[24-11-2020:19:16:04] p15868 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\client.py:430} INFO - CLIENT | New node initiated:: MAIN | europe | (Shard: None) 
[24-11-2020:19:16:55] p15868 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\player.py:279} INFO - PLAYER | Disconnected from voice channel:: None
[26-11-2020:09:15:20] p12428 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\websocket.py:85} ERROR - WEBSOCKET | Connection Failure:: Cannot connect to host 127.0.0.1:2333 ssl:default [The remote computer refused the network connection]
[26-11-2020:09:15:20] p12428 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\node.py:138} INFO - NODE | MAIN connected:: MAIN | europe | (Shard: None)
[26-11-2020:09:15:20] p12428 {C:\Users\Alberto\AppData\Local\Programs\Python\Python38-32\lib\site-packages\wavelink\client.py:430} INFO - CLIENT | New node initiated:: MAIN | europe | (Shard: None) 
